lib-src/luaexpat-1.2.0/src/lxplib.c

```diff
@@ -517,7 +529,11 @@ static int lxp_stop (lua_State *L) {
   return 1;
 }
 
+#if defined(LUAJIT_VERSION) && LUAJIT_VERSION > 20
+static const struct luaL_Reg lxp_meths[] = {
```

Comment:
Hey @chincheta0815, why do we need changes of this type where all we are doing is changing the name of the struct but not the contents/shape of the struct ?
IMHO the change would be smaller, and simpler with fewer #ifdefs (which is always a good thing)

Reply:
I totally agree with you, but I do not know how to account for a simple change in a lower/upper letter case change in a different way...

Reply:
ah okay,